README.md

```diff
@@ -108,6 +108,23 @@ ShopifyApp.configure do |config|
 end
 ```
 
+
+WebhooksManager
+---------------
+
+ShopifyApp can manage your apps webhooks for you (requires ActiveJob). Set which webhooks you require in the initializer:
+
+```ruby
+ShopifyApp.configure do |config|
+  config.webhooks = [
+    {topic: 'carts/update', address: 'example-app.com/webhooks', format: 'json'}
```

Comment:
> there isn't really any validation on what you set here. I think this should be fine though - most config type things don't really validate.

I'm not sure about this. It wouldn't be great if you typo'd the `topic` or `address` key names, but I guess the dev would notice quickly enough when CreationFailed gets raised. There's nothing we can do to prevent people from typoing the topic, since we don't want to include a full list of webhook topics in this gem, so I'd say just letting the dev run into `CreationFailed` would be fine here.

> I am also debating making json default do it doesn't need to be set here

This is a good idea. Personally, I would default it to `json` and not even mention that the format is an option.